datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -362,15 +363,17 @@ fn rountrip_aggregate() -> Result<()> {
             false,
         )?],
         // NTH_VALUE
-        vec![Arc::new(NthValueAgg::new(
-            col("b", &schema)?,
-            1,
-            "NTH_VALUE(b, 1)".to_string(),
-            DataType::Int64,
+        vec![udaf::create_aggregate_expr(
+            &AggregateUDF::new_from_impl(NthValueAgg::default()),
```

Comment:
we can just use `nth_value_udaf`

Reply:
Done in https://github.com/apache/datafusion/pull/11287/commits/5874f0f044eacd86b531e094d80af6285698a1b9